src/Symfony/Bridge/PhpUnit/bootstrap.php

```diff
@@ -20,10 +19,6 @@
 // Enforce a consistent locale
 setlocale(LC_ALL, 'C');
 
-if (!class_exists('Doctrine\Common\Annotations\AnnotationRegistry', false) && class_exists('Doctrine\Common\Annotations\AnnotationRegistry')) {
-    AnnotationRegistry::registerLoader('class_exists');
```

Comment:
To preserve BC, I think we should keep registering this, but add a check in the above if to not do it when 3.3 is in use (dunno which check exactly right now.)